spec/models/custom_button_spec.rb

```diff
@@ -325,6 +327,30 @@
           )
         end
       end
+
+      %i(invoke invoke_async).each do |method|
+        describe "##{method}" do
+          it "publishes CustomButtonEvents with multiple vms" do
+            Timecop.freeze(Time.now.utc) do
+              User.with_user(user) { custom_button.send(method, [vm, vm2, vm3], 'UI') }
+              expect(CustomButtonEvent.first.timestamp).to be_within(0.01).of(Time.now.utc)
+            end
+
+            expect(CustomButtonEvent.count).to eq(3)
+            expect(CustomButtonEvent.first).to have_attributes(
+              :source      => 'UI',
+              :target_id   => vm.id,
+              :target_type => 'VmOrTemplate',
+              :type        => 'CustomButtonEvent',
+              :event_type  => 'button.trigger.start',
+              :user_id     => user.id
+            )
+            expect(CustomButtonEvent.first[:full_data]).to include(
```

Comment:
Can this be merged into the expectation above?
`:full_data => a_hash_including(:automate_entry_point => "/SYSTEM/PROCESS/Request")`